packages/ui/src/redux/slices/settings-dialog.ts

```diff
@@ -17,12 +17,12 @@ const settingsDialogSlice = createSlice({
     switchSettingsDialogScreen: (state: Draft<SettingsDialog>, action: PayloadAction<SettingsDialogScreen>) => {
```

Comment:
Since we're inside the `settingsDialog` state, so we could shorten the `switchSettingsDialogScreen` a bit so it wont be too long, maybe `switchScreen` or `setScreen` should be clear enough. Same for the property `settingsDialogScreen` -> `screen`, any thoughts?